docs/source/examples.rst

```diff
@@ -93,31 +93,53 @@ Pre and Post Processing
 Case studies
 ------------
 
+Photonic Integrated Circuit Components
+~~~~~~~~~~~
 .. toctree::
     :maxdepth: 1
 
-    notebooks/Near2FarSphereRCS
-    notebooks/PlasmonicNanoparticle.ipynb
-    notebooks/PlasmonicYagiUdaNanoantenna
-    notebooks/GratingEfficiency
-    notebooks/BiosensorGrating.ipynb
-    notebooks/BraggGratings.ipynb
-    notebooks/DistributedBraggReflectorCavity
-    notebooks/HighQ_Si
-    notebooks/HighQ_Ge
-    notebooks/DielectricMetasurfaceAbsorber.ipynb
-    notebooks/GradientMetasurfaceReflector
-    notebooks/Metalens
-    notebooks/Bandstructure.ipynb
-    notebooks/OptimizedL3.ipynb
     notebooks/YJunction.ipynb
     notebooks/MMI_1x4
     notebooks/WaveguideCrossing
     notebooks/WaveguideSizeConverter
+    notebooks/BraggGratings.ipynb
     notebooks/EulerWaveguideBend
     notebooks/PolarizationSplitterRotator
     notebooks/8ChannelDemultiplexer
     notebooks/GratingCoupler
     notebooks/RingResonator
     notebooks/THzDemultiplexerFilter
-    notebooks/ZonePlateFieldProjection
\ No newline at end of file
+    notebooks/PhotonicCrystalWaveguidePolarizationFilter
+
+Metamaterials, Grattings, and Other Periodic Structures
+~~~~~~~~~~~
+.. toctree::
+    :maxdepth: 1
+
+    notebooks/DielectricMetasurfaceAbsorber.ipynb
+    notebooks/GradientMetasurfaceReflector
+    notebooks/Metalens
+    notebooks/HighQ_Si
+    notebooks/HighQ_Ge
+    notebooks/GratingEfficiency
+    notebooks/BiosensorGrating.ipynb
+    notebooks/DistributedBraggReflectorCavity
+
+Photonic Crystals
+~~~~~~~~~~~
```

Comment:
I think this will only work properly if the underline is the same length as the title (

```
Photonic Crystals
~~~~~~~~~~~~~~~~~
```
same for the other subsections

Reply:
Thanks for the reminder! I didn't notice that requirement. Updated in the current version.